ding/torch_utils/data_helper.py

```diff
@@ -8,7 +8,32 @@
 import numpy as np
 import torch
 
+def one_hot_embedding(labels, num_classes):
```

Comment:
这里如果是一个episode开始时，action的one-hot会设置为全零，在data里这个动作是存为int(action_shape)的，可能需要下面的one_hot_embedding_none()来特殊处理

Reply:
OK, for this case, I think you can add this part into the original `one_hot` function. And I recommend use `-1` rather than `None` to label this special element, it can be arranged into a tensor.

Reply:
好的